packages/facebook-config-ui/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: "Blaze configuration templates for Facebook OAuth.",
```

Comment:
I'm just copying the existing package.js. It does work and if we add name, we should add it in other packages too.